integration/kubernetes/runtimeclass_workloads/pod-footloose.yaml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: footfed
```

Comment:
Supreme nit. Should we call this `footclr` ? ;-P

Reply:
@ganeshmaharaj changes applied